### ISSUE TITLE
Error: No such host is known

### ISSUE THREAD

**Issue by FSou1** [denodrivers/deno_mongo#168]
When I run this code:

```
import { MongoClient } from "https://deno.land/x/mongo@v0.21.0/mod.ts";

const client = new MongoClient();
await client.connect("mongodb+srv://admin:ReelmKTm3pr9f9MG@cluster0.vsk29.mongodb.net/test?retryWrites=true&w=majority");
```

I get the following error:

```
error: Uncaught (in promise) Error: No such host is known. (os error 11001)
    at processResponse (deno:core/core.js:223:11)
    at Object.jsonOpAsync (deno:core/core.js:240:12)
    at async Object.connect (deno:runtime/rt/30_net.js:224:13)
    at async MongoClient.connect (client.ts:41:14)
    at async mod.ts:4:1
```

The same host is normally connected with Node.js and the `mongodb` npm package.

**Comment by manyuanrong** [comment]
Change `mongodb+srv://` to `mongodb://`

**Comment by FSou1** [comment]
> Change `mongodb+srv://` to `mongodb://`

Does it work for you?

**Comment by FSou1** [comment]
As I could see, this is a matter of the [new URL format](https://docs.mongodb.com/manual/reference/connection-string/#dns-seed-list-connection-format), which has been introduced as a part of the 3.6 version

**Comment by DryDemon** [comment]
Any solution or way to find a usable link?

**Comment by femave** [comment]
in v0.12.1 looks like all working well....

**Comment by DryDemon** [comment]
For information https://www.clever-cloud.com/en/ propose a mongo database tat works with the last version

**Comment by land47** [comment]
> in v0.12.1 looks like all working well....

this works for me.

**Comment by manyuanrong** [comment]
closed by #198
